Include/pyport.h

```diff
@@ -406,7 +406,7 @@ extern "C" {
 #endif
 
 /* get and set x87 control word for VisualStudio/x86 */
-#if defined(_MSC_VER) && defined(_M_IX86) /* x87 only supported in x86 */
+#if defined(_MSC_VER) && !defined(_WIN64) && !defined(_M_ARM) /* x87 not supported in 64-bit or ARM */
```

Comment:
@zooba Do you think this change and the one in pythonrun.h are related enough to the ctypes changes?
Or should they be seperated into a different PR?

Reply:
They're fine here I think